espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/GetLocationInView.kt

```diff
@@ -16,18 +16,15 @@
 
 package io.appium.espressoserver.lib.handlers
 
-import io.appium.espressoserver.lib.handlers.exceptions.AppiumException
 import io.appium.espressoserver.lib.model.AppiumParams
-import io.appium.espressoserver.lib.model.EspressoElement
 import io.appium.espressoserver.lib.model.Location
+import io.appium.espressoserver.lib.model.EspressoElement
 import io.appium.espressoserver.lib.model.ViewElement
 
 class GetLocationInView : RequestHandler<AppiumParams, Location> {
 
-    @Throws(AppiumException::class)
-    override fun handleInternal(params: AppiumParams): Location {
```

Comment:
do we also want to have handleCompose here?

Reply:
Perhaps yes, but I don't see the same method in the compose element so this PR does not have it so far.